offgridplanner/optimization/processing.py

```diff
@@ -234,16 +243,54 @@ def collect_supply_opt_json_data(self):
             "energy_system_design": energy_system_design,
         }
 
+        response = requests.get("http://127.0.0.1:5001/supply_schema", timeout=10)
```

Comment:
```suggestion
        response = requests.get("{SIM_API_HOST}/supply_schema", timeout=10)
```
This should just point to the simulation API host from the environment variables, one can then just change it to the localhost there if running the server locally